lib/commands/install.ts

```diff
@@ -0,0 +1,83 @@
+///<reference path="../.d.ts"/>
+"use strict";
+
+import path = require("path");
+
+export class InstallCommand implements ICommand {
+	private _projectFilePath: string;
+	
+	constructor(private $fs: IFileSystem,
+		private $errors: IErrors,
+		private $logger: ILogger,
+		private $options: IOptions,
+		private $injector: IInjector,
+		private $staticConfig: IStaticConfig) { }
+		
+	public enableHooks = false;
+		
+	public allowedParameters: ICommandParameter[] = [];
+	
+	public execute(args: string[]): IFuture<void> {
+		return (() => {
+			let projectFilePath = this.getProjectFilePath(args[0]);
+			let projectData = this.getProjectData(projectFilePath).wait();
+			let projectName = projectData.id.split(".")[2];
```

Comment:
You should validate that projectData has id. I've tried removing it from my package.json and the result is:
```
$ tns install .
Cannot read property 'split' of undefined
```